### ISSUE TITLE
Prices in media optimization

### ISSUE THREAD

**Issue by uomodellamansarda** [google/lightweight_mmm#186]
Reading the documentation on [Media Optimization](https://lightweight-mmm.readthedocs.io/en/latest/api.html#lightweight_mmm.optimize_media.find_optimal_budgets), I have one doubt.
When we are trying to optimize the media spend what is the argument "prices". 

_"prices – An array with shape (n_media_channels,) for the cost of each media channel unit"_

My question is:

1. Is the total cost of the media during the model training? Eg. I build a MM Model on a time period of two years with a total budget on YouTube of 250000£ and this is the cost used as input by the optimization object?

`lightweight_mmm.optimize_media.find_optimal_budgets(n_time_periods: int, media_mix_model: [lightweight_mmm.lightweight_mmm.LightweightMMM](https://lightweight-mmm.readthedocs.io/en/latest/api.html#lightweight_mmm.lightweight_mmm.LightweightMMM), budget: Union[float, int], prices: jax.Array, extra_features: Optional[jax.Array] = None, media_gap: Optional[jax.Array] = None, target_scaler: Optional[[lightweight_mmm.preprocessing.CustomScaler](https://lightweight-mmm.readthedocs.io/en/latest/api.html#lightweight_mmm.preprocessing.CustomScaler)] = None, media_scaler: Optional[[lightweight_mmm.preprocessing.CustomScaler](https://lightweight-mmm.readthedocs.io/en/latest/api.html#lightweight_mmm.preprocessing.CustomScaler)] = None, bounds_lower_pct: Union[float, jax.Array] = 0.2, bounds_upper_pct: Union[float, jax.Array] = 0.2, max_iterations: int = 200, solver_func_tolerance: float = 1e-06, solver_step_size: float = 1.4901161193847656e-08, seed: Optional[int] = None)→ scipy.optimize._optimize.OptimizeResult`
